qkernel/src/syscalls/sys_proxy.rs

```diff
@@ -103,6 +103,16 @@ pub fn SysProxy(task: &mut Task, args: &SyscallArguments) -> Result<i64> {
             );
             return Ok(ret);
             
+        }
+        ProxyCommand::CudaUnRegisterFatBinary => {
+            //error!("fatCubinHandle from the cudaproxy is {:x}", parameters.para1 as u64);
+            let ret = HostSpace::Proxy(
+                 ProxyCommand::CudaUnRegisterFatBinary,
```

Comment:
CudaUnregisterFatBinary, here R should be lowercase to r. Please change all the occurrences.